apps/entities/api/serializers.py

```diff
@@ -32,25 +26,18 @@ def get_serialized_end_date(self, instance: 'Entity'):
         return instance.end_date.serialize() if instance.end_date else None
 
 
-class EntitySerializer(SearchableModelSerializer):
+class EntityModelSerializer(TypedModuleSerializerDrf):
```

Comment:
Per https://github.com/ModularHistory/modularhistory/pull/657/files#r702437998 , maybe this one could be `EntityDrfSerializer`.

Reply:
I added the word `Model` to reflect that they're backed by DRF's `ModelSerializer`, should I still rename to DRF?

Reply:
The current naming makes sense in context of DRF's `ModelSerializer`, but I think I would prefer the `DrfSerializer` suffix for extra clarity and consistency, considering that `ModelSerializer` doesn't necessary evoke DRF (at least in my mind) and we include `Drf` in the names of our parent classes for these serializers. Would you be OK with that?